README.md

```diff
@@ -119,10 +119,12 @@ docker exec -it cont-<NAME> bash
 
 ## Releasing updates
 
-Travis will push an image to DockerHub with every successful run.
-If it's tagged (ie `git tag v0.0.x && git push origin --tags`),
-then that version number will be pushed to DockerHub, and `latest`
-will be updated as well.
+The CI workflow automatically pushes new builds to Dockerhub. All that's necessary is to bump the version number and push to master:
+
+```
+bumpversion [patch,minor,major]
+git push origin master --tags
+```
```

Comment:
What do I have to do, when my pull request has been approved? Do I
- merge branch to master
- run `bumpversion 0.7.4` <- That does not seem right
- run `git push origin master --tags`?

Reply:
Good question. The procedure when a pull request is approved is to just merge with master.

If you'd like to issue a new release, then from the master branch do the following:

1. Update the CHANGELOG to add the latest version.
2. do `bumpversion [patch|minor|major]`
3. `git push origin master --tags`.

I updated the REAMDE to be more clear about this. If you think we can do this more efficiently, I'm happy to switch to another system.